demisto_sdk/__main__.py

```diff
@@ -3546,6 +3546,65 @@ def run_unit_tests(
     sys.exit(unit_test_runner(file_paths, verbose))
 
 
+@main.command(short_help="Setup integration environments")
+@click.option(
+    "-i",
+    "--input",
+    type=PathsParamType(
+        exists=True, resolve_path=True
+    ),  # PathsParamType allows passing a list of paths
+    help="A list of content packs/files to validate.",
+)
+@click.option(
+    "--create-virtualenv",
+    is_flag=True,
+    default=False,
+    help="Create a virtualenv for the environment",
+)
+@click.option(
+    "--overwrite-virtualenv",
+    is_flag=True,
+    default=False,
+    help="Overwrite existing virtualenvs. Use with the create-virtualenv flag",
+)
+@click.option(
+    "--secret-id",
+    help="Secret ID, to use with PANW's Google Secret Manager instance. (internal use only)",
```

Comment:
if its for internal use, maybe we can hide this parameter from customers?

Reply:
also, how would customers use it?

Reply:
This can be used by anyone that uses google secret manager.
To use it, we need to configure the env variable DEMISTO_SDK_GCP_PROJECT_ID, which can point to a custom project.
I changed the docstring.